docs/content/engines/flink.md

```diff
@@ -210,3 +210,19 @@ All Flink data types are supported, except that
 
 * `MULTISET` is not supported.
 * `MAP` is not supported as primary keys.
+
+## Use Flink Managed Memory
+In order to improve the buffer pool for writers in Paimon tasks.Paimon tasks can create memory pools based on Executor Memory which will be managed by Executor, such as Managed Memory in Flink TaskManager. It will improve the stability and performance of sinks by managing writer buffers for multiple tasks through Executor.
+
+The following properties can be set if using the flink managed memory:
+
+| Option    | Default | Description                                                                                                                                                                    |
+|------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| sink.use-managed-memory-allocator | false | If true, flink sink will use managed memory for merge tree; otherwise, it will create an independent memory allocator.                   |
+| sink.managed.writer-buffer-memory | 256M  | Weight of writer buffer in managed memory, Flink will compute the memory size, for writer according to the weight, the actual memory used depends on the running environment. |
```

Comment:
"Weight of writer buffer"? This is not the real memory size? Could you provide a link to Flink's document about how managed memory are allocated to different operators?

Reply:
Thank you for the review. I have made some modifications